Automattic/wp-calypso
# VideoPress User Intent: randomize Intent items, keep "other" always last

Related to # https://github.com/Automattic/greenhouse/issues/1835

## Proposed Changes

* randomize the order of the Intent options
* keep "Other" always last

## Testing Instructions



* `yarn && yarn start`
* visit `/setup/videopress`
* notice the order of the items
* press refresh
* notice the order of the items has changed
* "Other" should always be last
* Items shouldn't re-order themselves during page load, the order should stay consistent between react re-renders due to state change

## Pre-merge Checklist
